src/main/java/seedu/address/model/person/MoneyOwed.java

```diff
@@ -68,6 +68,21 @@ public boolean userOwesMoney() {
         return (moneyOwed < 0);
     }
 
+    /**
+     * Returns the amount of money owed.
+     */
+    public Float getAmount() {
+        return this.moneyOwed;
+    }
```

Comment:
Do you think this can be removed since moneyOwed attribute is public?

Reply:
hmm im trying to use methods instead of accessing attributes directly because of OOP principles